src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--date--1 calendar__day--start-mon"></div>
```

Comment:
why do you need? it only changes first day number
calendar__day--date--1
<img width="1137" alt="image" src="https://user-images.githubusercontent.com/102411130/225842589-a7c8dd05-e3f1-4825-a029-9089c4971095.png">

but it should change month length
<img width="631" alt="image" src="https://user-images.githubusercontent.com/102411130/225842381-317ff9da-6d2d-48e4-96f2-6456f811a0c6.png">
<img width="777" alt="image" src="https://user-images.githubusercontent.com/102411130/225842751-db84d5f0-dcfc-49a5-96fc-0cae641c027c.png">

Reply:
fix this point